src/mediasession-helper.js

```diff
@@ -128,3 +133,22 @@ export function parseFavicon(doc) {
     return;
   }
 }
+
+/**
+ * @private
+ */
+function validateMetadata(metadata) {
+  // Ensure src of artwork has valid protocol
+  if (metadata && metadata.artwork) {
+    if (isArray(metadata.artwork)) {
+      metadata.artwork.forEach(artwork => {
+        if (artwork) {
+          const src = isObject(artwork) ? artwork.src : artwork;
+          user().assert(isProtocolValid(src));
+        }
+      });
+    } else {
+      user().assert(isProtocolValid(metadata.artwork));
```

Comment:
Throw here, artwork can't be just a `string`, has to be an array of MediaImages [https://wicg.github.io/mediasession/#mediametadata](https://wicg.github.io/mediasession/#mediametadata)